core/tests/bank_forks.rs

```diff
@@ -206,24 +257,34 @@ mod tests {
         }
     }
 
-    #[test]
-    fn test_concurrent_snapshot_packaging() {
+    fn run_test_concurrent_snapshot_packaging(snapshot_version: SnapshotVersion) {
         solana_logger::setup();
 
         // Set up snapshotting config
-        let mut snapshot_test_config = setup_snapshot_test(1);
+        let mut snapshot_test_config = SnapshotTestConfig::new(snapshot_version, 1);
 
         let bank_forks = &mut snapshot_test_config.bank_forks;
         let accounts_dir = &snapshot_test_config.accounts_dir;
         let snapshots_dir = &snapshot_test_config.snapshot_dir;
         let snapshot_config = &snapshot_test_config.snapshot_config;
         let mint_keypair = &snapshot_test_config.genesis_config_info.mint_keypair;
         let genesis_config = &snapshot_test_config.genesis_config_info.genesis_config;
+        let snapshot_version = bank_forks
```

Comment:
Like [this](https://github.com/solana-labs/solana/pull/10536/files#r442013709), why is this needed?

Reply:
It's not, changes were solely to parameterize snapshot version.
I've tidied up the bank forks tests.